setup.py

```diff
@@ -24,7 +24,8 @@
 
 INSTALL_REQUIRES = [
     'metomi-rose>=2.0a1',
-    'cylc-flow>=8.0a3'
+    'cylc-flow>=8.0a3',
+    'nest_asyncio'
```

Comment:
Hadn't spotted this earlier, I can't see any `async` statements here yet so I wouldn't have thought that this is necessary?

Reply:
I'll try removing it, but the rose functions involved use asyncio.